pep-9999.txt

```diff
@@ -0,0 +1,212 @@
+PEP: 9999
+Title: Support for indexing with keyword arguments
+Version: $Revision$
+Last-Modified: $Date$
+Author: Stefano Borini, Jonathan Fine
+Sponsor: 
+Discussions-To: python-ideas@python.org
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 24-Aug-2020
+Python-Version: 3.10
+Post-History: 
+Resolution: 
+
+Abstract
+========
+
+This PEP proposed extending python to allow keyword-like arguments to be
+accepted during indexing operations. Notations in the form ``a[K=3, R=2]``
+would become legal syntax.  A final strategy will be proposed in terms of
+semantics and implementation.
+
+This PEP is a rework and expansion of PEP-0472, where an extension of the
```

Comment:
Here and elsewhere, numbered PEPs are usually referred to without a hyphen or leading zeroes:

```suggestion
This PEP is a rework and expansion of PEP 472, where an extension of the
```